lib/ranked-model/ranker.rb

```diff
@@ -69,6 +70,19 @@ def update_rank! value
           update_all(ranker.column => value)
       end
 
+      def insert_after(previous_ranker)
+        if previous_ranker.instance.class != instance.class
```

Comment:
Need more inputs on this.

I know that there is already a method `validate_ranker_for_instance` which validates the instance. However, this is just to check if both are the same instance class to avoid passing objects of two different class with similar ranked-model definitions in them.

Reply:
Yes, I think that is definitely a better way of doing this. I'm thinking of taking this up in a separate PR.